requirements.txt

```diff
@@ -1,2 +1,3 @@
 mako
 PyYAML
+nose
```

Comment:
Ah, gotcha. I would probably rather keep them in one file; it may be beneficial for people to have the test dependencies if they're cloning the repository, since it may be good for them to run tests on their code before submitting.

I'll add a comment in the file :)